src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -30,6 +30,7 @@
         "symfony/stopwatch": "~2.3|~3.0.0",
         "symfony/templating": "~2.1|~3.0.0",
         "symfony/translation": "~2.8",
+        "doctrine/cache": "~1.0",
```

Comment:
it is used for the annotation reader (previously, the FileCachedReader of ``doctrine/annotations`` was used by default, but it changed because the class is deprecated)